packages/utils/src/uuid.ts

```diff
@@ -1,5 +1,28 @@
-import { v1 as uuid } from 'uuid';
+import Long from 'long';
+import { parse, v4 as uuid } from 'uuid';
 
 export type UUID = string;
 
-export const create = (): UUID => uuid();
+export interface UUIDMsbLsb {
```

Comment:
Nit: I think I'd return these as numbers and let the caller decide to stringify them as necessary.